README.md

```diff
@@ -20,11 +20,25 @@ $ git clone --recursive https://github.com/pytorch/test-infra
 │       ├── download.pytorch.org
 │       └── metrics.pytorch.org
 ├── stats                                # CI related stats commited automatically by a bot
-└── tools                                # Tools and scripts
-    ├── clang-tidy-checks
-    └── scripts
+├── tools                                # Tools and scripts
+|   ├── clang-tidy-checks
+|   └── scripts
+└── torchci                              # Code for hud.pytorch.org and our pytorch bots which run there   
+    └── pages 
 ```
 
+## Setting up your Dev environment to locally run hud.pytorch.org
+1. Install yarn:
+    `brew install yarn`
+2. `cd torchci` and install dependencies with `yarn install`
+2. Setup your environment variables
+
+    a. Copy `torchci/.env.example` to `torchci/.env.local` to create a local copy of your envionrmnet variables.  This will NOT be checked into git
+
+    b. For every environment setting defined in there, copy over the corresponding value [from Vercel](https://vercel.com/torchci/torchci/settings/environment-variables) (this requires access to our Vercel deployment)
+    
+3. From `torchci` run `yarn dev` to start the dev server.  The local endpoint will be printed on the console, it'll most likely be `http://localhost:3000`
```

Comment:
Nit: extra space before "The"

Reply:
You can also do any of the other comands in package.json to help you develop 
<img width="331" alt="image" src="https://user-images.githubusercontent.com/34172846/182182594-f6a81c59-a1a6-48d0-84f6-c59f6a6aa316.png">
like build to check for issues, lint, etc.

Reply:
> Nit: extra space before "The"

Good eye. My 11th grade English teacher enforced those double spaces after every period. I might need to unlearn that 19 year old habit :P

https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period#:~:text=If%20you%20learned%20to%20type,was%20uneven%20on%20a%20typewriter.

> You can also do any of the other comands in package.json to help you develop

This is sweet! Thanks, added that to the readme